pb/service.swagger.json

```diff
@@ -1343,49 +1388,61 @@
       "properties": {
         "currentTime": {
           "type": "string",
-          "format": "int64"
+          "format": "int64",
+          "title": "@gotags: yaml:\"-\""
```

Comment:
Can't we somehow prevent this from being generated for swagger?

Reply:
No. I tried but I was not success. We need to create a script for that to remove `gotags`.